src/components/TokenLogo/index.js

```diff
@@ -82,6 +84,7 @@ export default function TokenLogo({ address, networkInfo, header = false, size =
           style={{
             boxShadow: '0px 6px 10px rgba(0, 0, 0, 0.075)',
             borderRadius: '24px',
+            backgroundColor: 'white',
```

Comment:
Remove this background, we should use token raw image

Reply:
<img width="451" alt="image" src="https://user-images.githubusercontent.com/23650671/165010149-9ec9ca3b-e63f-4029-8686-f4f5d9f20582.png">
It fix this issue

Reply:
<img width="414" alt="image" src="https://user-images.githubusercontent.com/23650671/165010220-ce48a639-2a76-4d67-ad38-bbfd68d8ba36.png">
without

Reply:
keep raw version. If need, lets change image src from token list, because some tokens if we put a white background, it'll be ugly

Reply:
Actually, in this logic it is showing native token in svg format, not taking from tokenlist. It only affect those native token's svg